lib/talkable/configuration.rb

```diff
@@ -0,0 +1,31 @@
+module Talkable
+  class Configuration
+    attr_accessor :site_slug
+    attr_accessor :api_key
+    attr_accessor :server
+    attr_accessor :js_integration_library
+
+    def initialize
+      self.site_slug  = ENV["TALKABLE_SITE_SLUG"]
+      self.api_key    = ENV["TALKABLE_API_KEY"]
+      self.server     = "https://talkable.com"
+    end
+
+    def apply(config)
+      config.each do |key, value|
+        send("#{key}=", value) if respond_to?("#{key}=")
```

Comment:
`respond_to?` checks only public methods if you don't explicitly specify to look among private methods as well. But anyway `public_send` looks safer.